plugin-server/src/main/pluginsServer.ts

```diff
@@ -495,17 +495,31 @@ export async function startPluginsServer(
         if (capabilities.sessionRecordingBlobIngestionV2) {
             const hub = await setupHub()
             const postgres = hub?.postgres ?? new PostgresRouter(serverConfig)
+            const objectStorage = hub?.objectStorage ?? getObjectStorage(serverConfig)
```

Comment:
Hmmmmm does this mean that we are using the same config as the other recorder? We definitely don't want to use the same bucket etc 🤔

Reply:
We use that bucket for a few other things too, so I wanted to get something running quickly. I can take a look at creating a new bucket – creating the bucket should be easy, I'm not sure how access keys are handled in this case though.